Agreements/External/8_DUBS.txt

```diff
@@ -1,2 +1,2 @@
-FIR_COPX:*:*:VATRY:EIDW:*:London S8:DUBS:*:24000:VATRY10
+FIR_COPX:*:*:LEMGU:EIDW:*:London S8:DUBS:*:24000:LEMGU
```

Comment:
I say leave the first instance of VATRY as LEMGU - the controller doesn't see this on ES, but it means at least that the logic of the agreement relies on a routing via VATRY rather than LEMGU, which also means it will display the agreement for longer, beyond LEMGU.

FIR_COPX:*:*:VATRY:EIDW:*:London S8:DUBS:*:24000:LEMGU

Reply:
In fact, better idea:

`FIR_COPX:*:*:LEMGU:EIDW:*:London S8:DUBS:*:24000:LEMGU
FIR_COPX:*:*:TIBGA:EIDW:*:London S8:DUBS:*:24000:TIBGA
FIR_COPX:*:*:VATRY:EIDW:*:London S8:DUBS:*:24000:VATRY10`

As long as they're in this order, it will pick the first agreement the flightplan matches.